### PR TITLE
Fix transmission

### LINKED ISSUE #67: Add losses for transmissions for each connection
**Describe the solution you'd like**
Store losses for each transmission/region_from/region_to in database like distances.

- [ ] Add model
- [ ] Add repository 
- [ ] Add schemas
- [ ] Add endpoints
- [ ] Add in zip upload
- [ ] Add in zip download

### PR DESCRIPTION
**Pull request by jr-hartono**
Fixes #67 

### This PR ...
- remove loss_per_unit and reimplement transmission losses like distances
- add API endpoints for transmission distances and losses
- add tests for transmission distances and losses

-----

### Checklist

Checklist for Contributors
- [x] Unit tests provided? 
      Unit tests are needed for new features or to reproduce the bug solved by this PR.
- [ ] Documentation updated?
- [x] Unit tests successful
- [x] Code smells (flake8) checked 

Checklist for Reviewers
- [ ] Read PR description 
- [ ] Check linked issue
- [ ] Check CI/CD (build, automated tests, linting, preview build)
- [ ] Check documentation changes
- [ ] Add labels to PR

### CODE REVIEW

File: ensysmod/crud/energy_transmission_distance.py [hunk 1]
```diff
@@ -1,30 +1,68 @@
-from typing import List
+from typing import List, Optional
```
Comment (maurerle) [inline ensysmod/crud/energy_transmission_distance.py:1]: the more pythonic way is now to go for `value: int | list[int]` for example.
Making the list and optional obsolete.
This is available from 3.10 on, but we might like to change this project wide some time

File: tests/validators/test_region_to_or_ref_validator.py [hunk 2]
```diff
@@ -1,13 +1,9 @@
-from typing import Type, List, Tuple, Dict, Any
+from typing import Any, Dict, List, Tuple, Type
```
Comment (maurerle) [inline tests/validators/test_region_to_or_ref_validator.py]: you might want to use isort, to have consistent sorting of params

**Review by maurerle — APPROVED** [review]
Did not take a deep dive into the code or test it, but thats fine, you can merge!

**Comment by jr-hartono** [comment]
I forgot the tests for the validators. Don't merge yet.

File: ensysmod/crud/energy_transmission_distance.py (hunk 1 above)
Comment (jr-hartono) [reply ensysmod/crud/energy_transmission_distance.py:1]: Understood, I was also thinking of making the whole codebase more uniform by adding black and isort as dev dependencies.

**Pull request merged**